hassio/src/system/hassio-supervisor-info.ts

```diff
@@ -249,6 +273,28 @@ class HassioSupervisorInfo extends LitElement {
     });
   }
 
+  private async _unsupportedDialog(): Promise<void> {
+    const resolution = await fetchHassioResolution(this.hass);
+    await showAlertDialog(this, {
+      title: "You are running an unsupported installation",
+      text: html`Below is a list of issues found with your installation, click
+        on the links to learn how you can resolve the issues. <br /><br />
+        ${resolution.unsupported.map(
+          (issue) => html`
+            <li>
+              <a
+                href="${documentationUrl(this.hass, ISSUES[issue].url)}"
```

Comment:
Should we check if the issue exists in `ISSUES`? This requires the frontend and backend to be aligned perfect.